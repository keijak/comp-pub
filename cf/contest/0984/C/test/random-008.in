20
138 150 47
18 115 56
96 184 11
174 67 193
88 175 173
189 184 104
133 87 169
64 67 152
41 200 178
26 71 115
146 124 125
163 146 7
125 32 15
89 59 117
20 167 145
170 97 199
148 6 19
84 43 191
0 44 52
29 141 131
