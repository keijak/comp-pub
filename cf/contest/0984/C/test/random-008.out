Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Infinite
Finite
Infinite
